app/coffee/WebApiManager.coffee

```diff
@@ -24,7 +24,7 @@ module.exports = WebApiManager =
 		}, (error, response, data) ->
 			return callback(error) if error?
 			if 200 <= response.statusCode < 300
-				callback null, data?.project, data?.privilegeLevel
+				callback null, data?.project, data?.privilegeLevel, data?.isRestrictedUser
```

Comment:
Maybe out of scope, but I don't like this `data?` pattern here. If there's no data then surely this is an error, and it'll look like garbage if/when we decaf it.

Reply:
good point, we should bail out early with an error if we don't get any data.